### ISSUE TITLE
dasht-server is inserting encoded apostrophes that (neo)vim-dasht can't decode

### ISSUE THREAD

**Issue by snoblenet** [sunaku/dasht#25]
`dasht-server` is converting single straight quotes into encoded apostrophes:

https://github.com/sunaku/dasht/blob/master/bin/dasht-server-http#L57

'Vanilla' `dasht` in the shell knows how to handle these. For example, you can get:

```
xdescribe                            Jasmine method
Describing Hooks                       Mocha guide
```
However, `vim-dasht` on Neovim does not. Instead, you get:
```
&apos;xdescribe&apos;                            Jasmine method
&apos;Describing Hooks&apos;                       Mocha guide
```
I can think of several ways of fixing this, but I'm raising the issue on this repo (vs on `vim-dasht`) as the simplest solution surely would be to just not encode those straight single quotes. 

Thoughts?

**Comment by snoblenet** [comment]
BTW this behaviour has stopped. I am not sure why. Feel free to close this issue if it suits.

**Comment by snoblenet** [comment]
I tell a lie. The problem is still there. It's just there is no longer an apostrophe at the start and end of each entry. The apostrophes in the middle of entries are still escaped in this way however.

**Comment by sunaku** [comment]
Thanks for your report.  Unfortunately, I'm unable to reproduce it on my system. :disappointed_relieved: 

I can at least isolate the issue to the rendering of XML entities, which is performed by the w3m browser.

Which version of w3m do you have?  On my system, I have:

```sh
$ w3m -V
w3m version w3m/0.5.3+git20170102, options lang=en,m17n,image,color,ansi-color,menu,cookie,ssl,ssl-verify,external-uri-loader,nntp,ipv6,alarm,mark
```

**Comment by snoblenet** [comment]
Thanks @sunaku. I have:
```
$ w3m -V
w3m version w3m/0.5.3, options lang=en,m17n,color,ansi-color,mouse,menu,cookie,ssl,ssl-verify,external-uri-loader,w3mmailer,nntp,ipv6,alarm,mark
```

**Comment by sunaku** [comment]
Good news, I'm finally able to reproduce this issue using regular Vim 8.1 (instead of NeoVim). 🎉

**Comment by sunaku** [comment]
Looks like this is an issue with w3m itself: I'm not able to reproduce it with Lynx or Chrome browsers:

![screenshot](https://user-images.githubusercontent.com/9863/43048569-7fcba62a-8d9e-11e8-9ffe-93f8794ed1d6.png)

**Comment by sunaku** [comment]
Fixed now in commit ea5025b1b99ef13a55fb208c6e07bdbe69da3a27 💖  Thanks for reporting this bug. 👍
